lib/webtrit_adapter_web/controllers/api/v1/common_schema.ex

```diff
@@ -192,6 +192,11 @@ defmodule WebtritAdapterWeb.Api.V1.CommonSchema do
     OpenApiSpex.schema(%{
       type: :object,
       properties: %{
+        call_id: %Schema{
```

Comment:
As for me, this property must be required. But we need to check, is `call_id` relevant for all possible CDR Info variations.

Reply:
Unfortunately, this field is not necessary for Portabilling API

Reply:
Ok, understand.